src/analysis/sensitivity.jl

```diff
@@ -32,6 +32,51 @@ function ks_statistic(ks::ApproximateKSTest)::Float64
     return sqrt(n) * ks.δ
 end
 
+"""
+    _get_factor_spec(model_spec::DataFrame, factors::Vector{Symbol})
+
+Get model spec for specified factors.
+
+# Arguments 
+- `model_spec` : Model specification, as extracted by `ADRIA.model_spec(domain)` or from a `ResultSet`
+- `factors` : Factors considered for sensitivity analysis
+"""
+function _get_factor_spec(model_spec::DataFrame, factors::Vector{Symbol})
+    factors_to_assess = model_spec.fieldname .∈ [factors]
+    foi_attributes = Symbol[:fieldname, :ptype, :lower_bound, :upper_bound]
+    return model_spec[factors_to_assess, foi_attributes]
+end
+
+"""
+    _category_bins(S::Int64, foi_spec::DataFrame)
+
+Get number of bins for categorical variables.
+
+# Arguments
+- `S` : Number of bins
+- `foi_spec` : Model specification for factors of interest
+"""
+function _category_bins(S::Int64, foi_spec::DataFrame)
+    max_bounds = maximum(foi_spec.upper_bound .- foi_spec.lower_bound)
+    return round(Int64, max(S, max_bounds))
+end
+
+"""
+    _get_cat_quantile(foi_spec::DataFrame, factor_c::String, steps::Vector{Float64})
+
+Get quantile for categorical variable, factor_c.
```

Comment:
Re the variable name, the larger point here is that the description in the docstring should focus on what the function does. It is very rare for a variable to be called out like this.